integration_tests/pyproject.toml

```diff
@@ -12,14 +12,16 @@ flake8 = "^3.8.4"
 black = "^20.8b1"
 flake8-black = "^0.2.1"
 flake8-isort = "^4.0.0"
+grpcio = "^1.42.0"
+grpcio-tools = "^1.42.0"
 pep8-naming = "^0.11.1"
 pytest-github-actions-annotate-failures = "^0.1.1"
-protobuf = "^3.13.0"
-grpcio = "^1.33.2"
+protobuf = "3.19.4"
 PyYAML = "^5.3.1"
 python-dateutil = "^2.8.1"
-pystarport = { git = "https://github.com/crypto-com/pystarport.git", branch = "main" }
-chainlibpy = "^1.0.0"
+# chainlibpy use version 0.2.3 pystarport
+pystarport = "=0.2.3"
+chainlibpy = "2.2.0"
```

Comment:
this version doesn't appear to be released on PyPI yet: https://pypi.org/project/chainlibpy/
+ the poetry.lock wasn't updated